python/cudf/cudf/pandas/_wrappers/pandas.py

```diff
@@ -744,7 +749,7 @@ def _get_eval_locals_and_globals(level, local_dict=None, global_dict=None):
     return local_dict, global_dict
 
 
-@register_proxy_func(pd.eval)
+@register_proxy_func(pd.core.computation.eval.eval)
```

Comment:
Curious, does this _need_ to be the exact location of `eval` instead of using the public `pd.eval`?

Similar to the register functions below e.g. `pd.api.extensions.register_dataframe_accessor`?

Reply:
I think that used to be the case when we first wrote it, but no longer so

Reply:
Ah OK. Would be nice if this could be avoided but not a big deal